### DISCUSSION TITLE
[Feature Request] - EcoFlow DELTA Pro 3 - New Product support

### DISCUSSION THREAD

**Discussion by cdrom1028** [tolwi/hassio-ecoflow-cloud#367] [Ideas]
Is the new DELTA Pro 3 going to be supported in the future?

**Comment by Nooshie** [comment]
I really hope this gets added soon =D

**Comment by asatron1** [comment]
https://github.com/tolwi/hassio-ecoflow-cloud/issues/270

Was there already an idea here?
